lib/printer.js

```diff
@@ -1155,6 +1155,29 @@ function genericPrintNoParens(path, options, print) {
 
         return concat(parts);
 
+    case "TemplateElement":
+        return n.value.raw;
```

Comment:
This technically needs to be `fromString(n.value.raw, options)`, since the rest of the return statements in this function return `Lines` objects.